lib/active_merchant/billing/credit_card.rb

```diff
@@ -49,8 +49,10 @@ module Billing #:nodoc:
     class CreditCard < Model
       include CreditCardMethods
 
-      cattr_accessor :require_verification_value
+      class_attribute :require_verification_value
```

Comment:
@jnormore this was necessary so sub-classes (i.e. `NetworkTokenizedCreditCard` could define their own required fields without changing their parent class's settings. Without this, when `NetworkTokenizedCreditCard` sets `self.require_verification_value = false` it would cause `CreditCard` to not require a CVV as well.

Regarding multi-threaded environments, would an instance variable based approach be preferred?

```ruby
class CreditCard
  self << class
    attr_accessor :require_verification_value
  end
end
```

I believe this is thread-safe while accomplishing the [intended function](https://coderwall.com/p/xhcmbg/cattr_accessor-vs-class_attribute).